### PR TITLE
Increase timeouts for service tests.

### PR DESCRIPTION
**Pull request by brendandburns**
@quinton-hoole

### CODE REVIEW

File: test/e2e/service.go [hunk 1]
```diff
@@ -808,7 +808,7 @@ var _ = Describe("Services", func() {
 })
 
 func waitForLoadBalancerIngress(c *client.Client, serviceName, namespace string) (*api.Service, error) {
-	const timeout = 4 * time.Minute
+	const timeout = 20 * time.Minute
```
Comment (quinton-hoole) [inline test/e2e/service.go]: nit: Please add comment explaining the ridiculously long timeout, with internal reference to b/21807001

File: test/e2e/service.go [hunk 2]
```diff
@@ -826,7 +826,7 @@ func waitForLoadBalancerIngress(c *client.Client, serviceName, namespace string)
 }
 
 func waitForLoadBalancerDestroy(c *client.Client, serviceName, namespace string) (*api.Service, error) {
-	const timeout = 4 * time.Minute
+	const timeout = 10 * time.Minute
```
Comment (quinton-hoole) [inline test/e2e/service.go]: nit: Same as above.

**Comment by quinton-hoole** [comment]
LGTM barring comment nits.

File: test/e2e/service.go (hunk 1 above)
Comment (brendandburns) [inline test/e2e/service.go]: done.

File: test/e2e/service.go (hunk 2 above)
Comment (brendandburns) [inline test/e2e/service.go]: done.

**Comment by brendandburns** [comment]
comments addressed, please re-check.

**Comment by quinton-hoole** [comment]
LGTM!

**Comment by brendandburns** [comment]
Shippable is a flake, re-running.

**Pull request merged**